nixos/modules/programs/foot/default.nix

```
@@ -0,0 +1,79 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.foot;
+
+  settingsFormat = pkgs.formats.iniWithGlobalSection { listsAsDuplicateKeys = true; };
+in
+{
+  options.programs.foot = {
+    enable = lib.mkEnableOption "foot terminal emulator";
+
+    package = lib.mkPackageOption pkgs "foot" { };
+
+    settings = lib.mkOption {
+      inherit (settingsFormat) type;
+      default = {
+        globalSection = { };
+      };
+      description = ''
+        Configuration for foot terminal emulator. Further information can be found in foot's man page foot.ini(5).
```

improve syntax slightly

```suggestion
        Configuration for foot terminal emulator. Further information can be found in {command}`man 5 foot.ini`.
```